Documentation.md

```diff
@@ -667,6 +690,19 @@ Not available
 </table>
 
 
+####WGSetValue
+Sets the currently selected view's value. Currently only useful for progress bar views.
+
+<table width="100%">
+<tr><th>Assembly</th><th>Applesoft</th></tr><tr><td><pre>
+X:		WGSetState
+A: 		new value
+</td><td>
+&SETV(V value)
```

Comment:
The AppleSoft parameter docs don't need a pseudotype. "&SETV(value)" is sufficient.